test/system/src/test/java/io/pravega/test/system/StreamsAndScopesManagementTest.java

```diff
@@ -0,0 +1,254 @@
+        controllerPerfStats.put("sealStreamMs", new ArrayList<>());
+        controllerPerfStats.put("deleteStreamMs", new ArrayList<>());
+        controllerPerfStats.put("deleteScopeMs", new ArrayList<>());
+        controllerPerfStats.put("updateStreamMs", new ArrayList<>());
+    }
+
+    @After
+    public void tearDown() {
+        streamManager.close();
+        ExecutorServiceHelpers.shutdown(executor);
+    }
+
+    /**
+     * This test executes a series of metadata operations on streams and scopes to verify their correct behavior. This
+     * includes the creation and deletion of multiple scopes both in correct and incorrect situations. Moreover, for
+     * each scope, the test creates a range of streams and tries to create, update, seal and delete them in correct and
+     * incorrect situations. The test also performs metadata operation on empty and non-empty streams.
+     */
+    @Test
+    public void testStreamsAndScopesManagement() {
+        // Perform management tests with Streams and Scopes.
+        for (int i = 0; i < TEST_ITERATIONS; i++) {
+            log.info("Stream and scope management test in iteration {}.", i);
+            testStreamScopeManagementIteration();
+        }
+
+        // Provide some performance information of Stream/Scope metadata operations.
+        for (String perfKey : controllerPerfStats.keySet()) {
+            log.info("Performance of {}: {}", perfKey, controllerPerfStats.get(perfKey).stream().mapToLong(x -> x).summaryStatistics());
+        }
+
+        log.debug("Scope and Stream management test passed.");
+    }
+
+    // Start region utils
+
+    private void testStreamScopeManagementIteration() {
+        for (int i = 0; i < NUM_SCOPES; i++) {
+            final String scope = "testStreamsAndScopesManagement" + String.valueOf(i);
+            testCreateScope(scope);
+            testCreateSealAndDeleteStreams(scope);
+            testDeleteScope(scope);
+        }
+    }
+
+    private void testCreateScope(String scope) {
+        assertFalse(streamManager.deleteScope(scope));
+        long iniTime = System.nanoTime();
+        assertTrue("Creating scope", streamManager.createScope(scope));
+        controllerPerfStats.get("createScopeMs").add(timeDiffInMs(iniTime));
+    }
+
+    private void testDeleteScope(String scope) {
+        assertFalse(streamManager.createScope(scope));
+        long iniTime = System.nanoTime();
+        assertTrue("Deleting scope", streamManager.deleteScope(scope));
+        controllerPerfStats.get("deleteScopeMs").add(timeDiffInMs(iniTime));
+    }
+
+    private void testCreateSealAndDeleteStreams(String scope) {
+        for (int j = 1; j <= NUM_STREAMS; j++) {
+            final String stream = String.valueOf(j);
+            StreamConfiguration config = StreamConfiguration.builder().scalingPolicy(ScalingPolicy.fixed(j)).build();
+
+            // Create Stream with wrong scope, which should not be successful.
```

Comment:
You can modify the `wrong scope` to `nonexistent scope.`

Reply:
Done.